complete/src/main/java/price/checker/entrypoint/ApiError.java

```diff
@@ -0,0 +1,20 @@
+package price.checker.entrypoint;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@ToString
+@AllArgsConstructor
+public class ApiError {

```

Comment:
https://projectlombok.org/features/Value  includes these three annotations i believe

Reply:
I used those three specific annotations because those were the annotations used in fuse for the exact same class